examples/gopher.nwk

```diff
@@ -0,0 +1,2 @@
+((h6, h7) h1, (h8, (h10, (h12, (h14, (h16, h17) h15) h13) h11) h9) h2) h0;
```

Comment:
This file was renamed to `gopher_louse_host.nwk`. Please remove this file use `gopher_louse_host.nwk` instead. Alternatively, you can rename `gopher_louse_host.nwk` to `gopher.nwk` and remove `gopher_louse_host.nwk`. If you choose to do this, you also have to modify tests that use `gopher_louse_host.nwk`).

Reply:
Removed.